mungegithub/mungers/approval-handler.go

```diff
@@ -64,6 +68,23 @@ func (*ApprovalHandler) EachLoop() error { return nil }
 // AddFlags will add any request flags to the cobra `cmd`
 func (*ApprovalHandler) AddFlags(cmd *cobra.Command, config *github.Config) {}
 
+// Returns associated issue, or 0 if it can't find any.
+// This is really simple, and could be improved later.
+func findAssociatedIssue(body *string) int {
+	if body == nil {
+		return 0
+	}
+	match := regexp.MustCompile(`#(\d+)`).FindStringSubmatch(*body)
```

Comment:
> We should check for a return value of nil here?

`FindStringSubmatch` returns a slice, I already check if the slice is empty (which is equivalent to checking if it's nil).